<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TInpaintProcessViewWidgetClass</class>
 <widget class="TProcessViewWidgetAbstract" name="TInpaintProcessViewWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>174</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TInpaintProcessViewWidgetAbstract</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="undoButton"/>
      </item>
      <item>
       <widget class="QToolButton" name="redoButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="penWidthGroupBox">
     <property name="title">
      <string>选择宽度：</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="1,0,1">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_23" native="true"/>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="penWidthSpinBox">
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
        </property>
        <property name="specialValueText">
         <string>最细宽度</string>
        </property>
        <property name="suffix">
         <string> px</string>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_24" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>修复区域选择方式：</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="manual">
        <property name="text">
         <string>手动选择</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="intelligent">
        <property name="text">
         <string>智能选择</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>修复方法：</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="NS">
        <property name="text">
         <string>Navier-Stokes</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="TELEA">
        <property name="text">
         <string>Alexandru Telea</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="undo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="processview.qrc">
     <normaloff>:/ProcessView/images/icons/MaterialSymbolsUndo.svg</normaloff>:/ProcessView/images/icons/MaterialSymbolsUndo.svg</iconset>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="redo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="processview.qrc">
     <normaloff>:/ProcessView/images/icons/MaterialSymbolsRedo.svg</normaloff>:/ProcessView/images/icons/MaterialSymbolsRedo.svg</iconset>
   </property>
   <property name="text">
    <string>重做</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TProcessViewWidgetAbstract</class>
   <extends>QWidget</extends>
   <header>tprocessviewwidgetabstract.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="processview.qrc"/>
 </resources>
 <connections/>
</ui>
